source/addonHandler/__init__.py

```diff
@@ -151,13 +151,11 @@ def initialize():
 	if config.isAppX:
 		log.info("Add-ons not supported when running as a Windows Store application")
 		return
-	loadState()
-	removeFailedDeletions()
-	completePendingAddonRemoves()
-	completePendingAddonInstalls()

```

Comment:
If we are concerned about breaking API changes here, this PR is blocked because these functions have been removed. I flag this as a concern as I'm not sure how addons like addonUpdater work.

Reply:
Addon Updater is not using any of these. In fact I don't think any  half serious add-on would since they modify content of addons folder and it is not safe to call them after add-ons were loaded. I don't think we should be voried about backwards compatibility for these removals.

Reply:
I agree that this is the case for `removeFailedDeletions`,  `completePendingAddonRemoves` and `completePendingAddonInstalls` but what about `loadState`? Should this get the same treatment as `saveState`? It's unlikely that either of these are used but they should be safe to call as far as I can tell.

Reply:
Done as of 38060bc00. I'm not convincet this has been necesary but better to be safe than sorry.